src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -189,14 +189,14 @@ RCLConsensus::Adaptor::propose(RCLCxPeerPos::Proposal const& proposal)
 
     prop.set_nodepubkey(valPublic_.data(), valPublic_.size());
 
-    auto signingHash = sha512Half(
-        HashPrefix::proposal,
-        std::uint32_t(proposal.proposeSeq()),
-        proposal.closeTime().time_since_epoch().count(),
-        proposal.prevLedger(),
-        proposal.position());
-
-    auto sig = signDigest(valPublic_, valSecret_, signingHash);
+    Serializer s;
+    s.add32(HashPrefix::proposal);
+    s.add32(std::uint32_t(proposal.proposeSeq()));
+    s.add32(proposal.closeTime().time_since_epoch().count());
+    s.add256(proposal.prevLedger());
+    s.add256(proposal.position());
```

Comment:
https://github.com/ripple/rippled/pull/2647/commits/ca2e49a60142c8d2b7b62055688f99cb997eab10 incorporates the changes from your commit, except I opted to replace `RCLCxPeerPos::signingData()` with a non-member function similar to the existing [proposalUniqueId](https://github.com/ripple/rippled/blob/11ab98cced69e1135a8a1e7ce55610273b46a4c0/src/ripple/app/consensus/RCLCxPeerPos.cpp#L74) instead of including more `ripple/` in the consensus code.